### PR TITLE
Feature/#744

### PR DESCRIPTION
**Pull request by schoicsiro**
Would you be able to get PR?

### CODE REVIEW

**Review by temi — COMMENTED** [review]
The code looks good. Please fix failing tests.

**Comment by schoicsiro** [comment]
All test cases are ok when I run test in my local because I changed attribute.text with index and reindexed
but my partial searching test cases will be failed until index has changed

**Comment by schoicsiro** [comment]
I fixed test cases without calling elastic search

File: src/integration-test/groovy/au/org/ala/profile/SearchServiceSpec.groovy [hunk 1]
```diff
@@ -1754,4 +1757,77 @@ class SearchServiceSpec extends BaseIntegrationSpec {
...
+        options.setSearchNsl(true)
+        options.setIncludeNameAttributes(false)
+        options.setHideStubs(true)
+
+        when:
+        Map qMap = service.buildTextSearch("dilan", options)
+
+        then:
+        qMap.findAll(it -> it.toString().contains("dilan")) != null
+        result.find { it.contains(profile1.scientificName) } != null
```
Comment (temi) [inline src/integration-test/groovy/au/org/ala/profile/SearchServiceSpec.groovy]: do not think result, profile1 or opus1 is needed. remove them.

File: src/integration-test/groovy/au/org/ala/profile/SearchServiceSpec.groovy [hunk 2]
```diff
@@ -1754,4 +1757,77 @@ class SearchServiceSpec extends BaseIntegrationSpec {
...
+
+        for (r in result) {
+            List texts = r.attributes.text
+            for (text in texts) {
+                isFind = text.contains(searchItem) || isFind
+            }
+        }
+
+        qMap.findAll(it -> it.toString().contains(searchItem)) != null
+        result && isFind
```
Comment (temi) [inline src/integration-test/groovy/au/org/ala/profile/SearchServiceSpec.groovy]: Same here result, profile, attributes are not needed. remove them. Only testing on qMap is needed.

**Review by temi — APPROVED** [review]

**Pull request merged**